downstream_projects.json

```diff
@@ -1,8 +1,9 @@
 {
   "packageDir": "./dist",
   "projects": {
-    "sample-app-angular": "https://github.com/ui-router/sample-app-angular.git",
+    "sample-app-angular": "https://github.com/lindolo25/sample-app-angular.git",
```

Comment:
What's the change here? Can it be put back as before?

Reply:
Hi @wawyed! Yes I added that change for test to run. There is a related update for sample-angular-app needed before this change can be reverted. https://github.com/ui-router/sample-app-angular/pull/790

Reply:
Could you explain why the change is necessary?

Reply:
Hi @wawyed!

The change in the `downstream_project.json` file is not necessary, and I have ready a commit to revert it. I temporally added it so the test action will run and complete with success.

The change for `sample-angular-app` is necessary! The `ui-view` is now a standalone component and it cannot be added as the bootstrap component of an NgModule. The change in this project is to create a component to use as the bootstrap component of the app. Checkout the changes here it will be clear https://github.com/ui-router/sample-app-angular/pull/790/files

As soon as https://github.com/ui-router/sample-app-angular/pull/790 is merged I will push the commit reverting the change in `downstream_projects.json`, all test will run again and pass.

Reply:
Does that mean that technically this could be a breaking change if someone for any reason had the uiview as a bootstrap component?

Reply:
Yes!

Reply:
Okay, so something to consider when releasing it as a major version in that case.

Reply:
That's now merged in sample-app-angular

Reply:
Yes, this release can be done as a major version! I will update the version and push both changes shortly. Thanks for all the help!

Reply:
Hi @wawyed, the updates you requested are ready for review!